src/ps.rs

```diff
@@ -271,26 +272,33 @@ pub fn create_snapshot(
 
     let mut writer = Writer::from_writer(io::stdout());
 
+    const VERSION: &str = env!("CARGO_PKG_VERSION");
+
     for ((user, job_id, command), job_info) in processes_by_job_id {
+        // FIXME this does not print "none" or "unknown"
```

Comment:
Customarily, add a link to the filed issue with the FIXME.

Reply:
Good suggestion. Will add.